src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        Optional<Driver> driver = driverDao.get(id);
+        return driver.orElseThrow(() -> new RuntimeException("Could not get driver. ID:" + id));
```

Comment:
```suggestion
        return driverDao.get(id).orElseThrow(() -> new NoSuchElementException("Could not get driver. ID:" + id));
```
I'd recommend making it this way